src/handler.py

```diff
@@ -15,8 +16,11 @@
 QA_LAMBDA_TRIGGERS = ['aqts-capture-trigger-QA-aqtsCaptureTrigger']
 SQL = "select count(1) from batch_job_execution where status not in ('COMPLETED', 'FAILED')"
 
+log_level = os.getenv('LOG_LEVEL', logging.ERROR)
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.DEBUG)
+logger.setLevel(log_level)
+
+cloudwatch_client = boto3.client('cloudwatch', os.getenv('AWS_DEPLOYMENT_REGION', 'us-south-10'))
```

Comment:
I think if the region is being defaulted, it should be to `us-west-2`, not the made-up `us-south-10`, which I typically find in our unit tests.

Reply:
Fixed.